_data/merchants.yml

```diff
@@ -176,6 +176,8 @@
       url: https://synntechgaming.com
     - name: All For Crypto
       url: https://allforcrypto.com
```

Comment:
In the same pull request, can we also move the All for Crypto to the top of the goods list to keep it alphabetical?

Reply:
I didn't notice it earlier. edited, thanks